pixels-daemon/src/main/java/io/pixelsdb/pixels/daemon/cache/CacheWorker.java

```diff
@@ -256,6 +258,44 @@ public void run()
                         }
                     }
                 });
+        // watcher for cache expand or shrink.
```

Comment:
Also update the docs (e.g., README.md), keep it consistent with the new zoned implementation. Make sure people can set up and use the cache following the instructions in the docs.